src/labels/CustomLabels.labels

```diff
@@ -7716,16 +7716,24 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <categories>Getting Started Page</categories>
         <language>en_US</language>
         <protected>true</protected>
-        <shortDescription>gsDaysRemainingInFreeTrial</shortDescription>
+        <shortDescription> Days remaining in the free trial label for the application status component</shortDescription>
         <value>Days Remaining in Free Trial</value>
     </labels>
+    <labels>
+        <fullName>gsEmailAddress</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Power of us email address</shortDescription>
+        <value>powerofus@salesforce.com</value>
```

Comment:
I'm curious why some URL's are hardcoded into the html file and other URL's (or emails) are handled as a custom label? There should be consistency throughout all of the Getting Started pages on how these are handled. Changes to the email address in the future can be made just as easily in either area.

Reply:
@dcaballeroSf What is your thought on this?

Reply:
@force2b done